resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,134 @@
+"""Reverse complement reads with Seqtk."""
+import os
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk:comp'
+    name = 'Reverse complement FASTQ'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image' : 'resolwebio/common:1.1.0'
```

Comment:
Any reason not to use the [latest version](https://hub.docker.com/r/resolwebio/common/tags) of `resolwebio/common`?